byron/chain/executable-spec/test/Cardano/AbstractSize/Properties.hs

```diff
@@ -0,0 +1,163 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.AbstractSize.Properties
```

Comment:
The tests here seem to similar to `Ledger.HasTypeReps.Properties`, why do we have them in two separate modules?

Reply:
Oh, now I see. Brain fart. These are in two separate packages. So what about giving them some consistent names? E.g.:

- `Cardano.Ledger.AbstractSize.Properties`
- `Cardano.Chain.AbstractSize.Properties`

?

Reply:
👍